test/unit/config/config-location.test.js

```diff
@@ -57,7 +57,7 @@ tap.test('when overriding the config file location via NEW_RELIC_HOME', (t) => {
     await fsPromises.rm(DESTDIR, { recursive: true, force: true })
 
     process.chdir(startDir)
-    await fsPromises.rmdir(NOPLACEDIR, { recursive: true })
+    await fsPromises.rm(NOPLACEDIR, { recursive: true })
```

Comment:
I'm curious about the conversion to `rm` here. `fsPromises` doesn't seem to have an `.rm` method in its latest API docs.

Reply:
Hm? https://nodejs.org/api/fs.html#fspromisesrmpath-options

`rmdir` is deprecated. This change was to silence a warning.